service/src/test/java/bio/terra/externalcreds/services/ProviderServiceTest.java

```diff
@@ -1114,4 +1114,127 @@ private ClientRegistration createClientRegistration(Provider provider) {
         .authorizationGrantType(AuthorizationGrantType.REFRESH_TOKEN)
         .build();
   }
```

Comment:
the new @TestComponent below is a copy-paste from the now-deleted `PassportProviderServiceTest`